agr_api/src/main/java/org/alliancegenome/api/controller/DiseaseController.java

```diff
@@ -37,354 +24,384 @@
-			throw new RestErrorException(error);
-		}
-	}
-
-	@Override
-	public Response getDiseaseAnnotationsRibbonDetailsDownload(String focusTaxonId, List<String> geneIDs, String termID, String filterSpecies, String filterGene, String filterReference, String diseaseTerm, String filterSource, String geneticEntity, String geneticEntityType, String associationType, String diseaseQualifier, String evidenceCode, String basedOnGeneSymbol, Boolean includeNegation, Boolean debug, String sortBy, String asc) {
-
-		LocalDateTime startDate = LocalDateTime.now();
-		Response.ResponseBuilder responseBuilder;
-		try {
-			JsonResultResponse<GeneDiseaseAnnotationDocument> response = getDiseaseAnnotationsRibbonDetails(focusTaxonId, geneIDs, termID, null, filterSpecies, filterGene, filterReference, diseaseTerm, filterSource, geneticEntity, geneticEntityType, associationType, diseaseQualifier, evidenceCode, basedOnGeneSymbol, includeNegation, debug, 150000, 1, sortBy, asc);
-			response.setHttpServletRequest(null);
-			response.calculateRequestDuration(startDate);
-			// translate all records
-			responseBuilder = Response.ok(translator.getAllRowsForGeneDiseaseAnnotations(response.getResults()));
-			responseBuilder.type(MediaType.TEXT_PLAIN_TYPE);
-			APIServiceHelper.setDownloadHeader(geneIDs.get(0), GENE, DISEASE, responseBuilder);
-		} catch (Exception e) {
-			e.printStackTrace();
-			log.error("Error: " + e);
-			RestErrorMessage error = new RestErrorMessage();
-			error.addErrorMessage(e.getMessage());
-			throw new RestErrorException(error);
-		}
-
-		return responseBuilder.build();
-	}
-
-	@Override
-	public Response getDiseaseAnnotationsBySpeciesDownload(List<String> speciesIDs, String diseaseID, String sortBy) {
-
-		if (CollectionUtils.isEmpty(speciesIDs)) {
-			speciesIDs = Arrays.stream(SpeciesType.values()).map(SpeciesType::getTaxonID).collect(Collectors.toList());
-		} else {
-			speciesIDs = speciesIDs.stream().map(SpeciesType::getTaxonId).collect(Collectors.toList());
-		}
-		List<DiseaseAnnotation> alleleAnnotations = new ArrayList<>();
-		List<DiseaseAnnotation> geneAnnotations = new ArrayList<>();
-		List<DiseaseAnnotation> modelAnnotations = new ArrayList<>();
-		speciesIDs.forEach(species -> {
-			alleleAnnotations.addAll(getDiseaseAnnotationsByAllele(diseaseID, Integer.MAX_VALUE, null, sortBy, null, null, species, null, null, null, null, null, null).getResults());
-			modelAnnotations.addAll(getDiseaseAnnotationsForModel(diseaseID, Integer.MAX_VALUE, null, sortBy, null, null, species, null, null, null, null, null, null).getResults());
-			geneAnnotations.addAll(getDiseaseAnnotationsByGene(diseaseID, Integer.MAX_VALUE, null, sortBy, null, species, null, null, null, null, null, null, null).getResults());
-		});
-		Response.ResponseBuilder responseBuilder = Response.ok(translator.getAllRowsForGenesAndAlleles(geneAnnotations, alleleAnnotations, modelAnnotations));
-		return responseBuilder.build();
-	}
+    // @Inject
```

Comment:
This controller was switched from tabs to spaces.

Reply:
Fixed.